pages/planetusermanual.md

```diff
@@ -5,3 +5,30 @@
 The [**Open Learning Exchange (OLE)**](http://ole.org) is committed to ensuring that everyone, especially children, has access to a high quality basic education. We work with nation-based entrepreneurs, educators, and practitioners committed to promoting highly effective and affordable learning systems that can benefit all of their people.  We believe this will lead to more productive lives, communities, and nations, and to the possibility of a peaceful world. 
 
 OLE’s mission is to ensure learning for all, not only universal basic literacy and numeracy, but also vital knowledge and skills acquisition in the areas of health, nutrition, community development. Since 2007, OLE International has helped establish Open Learning Communities for children, adolescents, and young adults, including those whose lives have been disrupted by violent conflicts.  Today there are OLE affiliates in in Asia, Africa, the Middle East, and Latin America.
+
+## The PLANET Learning System
+### What is the OLE PLANET Learning System (PLANET)?
+Personalized Learning And Networked Enabled Teams (PLANET) is a, multi-lingual learning system and library with thousands of educational Resources including books, videos, and educational games.  The system also includes software tools for content and student management, student evaluation and monitoring, and Courses creation.
```

Comment:
If you come across grammar mistakes or particular sections that need to be reworded in order to be clear, feel free to fix them up. In this case I am pretty sure that the comma after the `a` is not needed.